src/cpu_data.go

```diff
@@ -141,52 +143,63 @@ func NewCollector(handle func() (collect.AMDParams)) prometheus.Collector {
 		GPUDevId: prometheus.NewDesc(
 			prometheus.BuildFQName("amd", "", "gpu_dev_id"),
 			"AMD Params",// The metric's help text.
-			[]string{"gpu_dev_id", "productname"},// The metric's variable label dimensions.
+			[]string{"gpu_dev_id", "productname", "node_id", "pod"},// The metric's variable label dimensions.
```

Comment:
argument "pod" is not working for any of the APIs when we try to query. Please check
![image](https://github.com/amd/amd_smi_exporter/assets/105346952/87406864-00eb-4482-9f63-9616815f66f3)

Reply:
Query with argument "node_id" also gives  "Empty Query result"
![image](https://github.com/amd/amd_smi_exporter/assets/105346952/fc12345f-6ac0-4998-94c3-1d52f2c357fd)